docs/source/tutorials/base/how_to_train/detection.rst

```diff
@@ -0,0 +1,385 @@
+Object Detection model
+======================
+
+This tutorial reveals end-to-end solution from installation to model export and optimization for object detection task on a specific example.
+On this page we show how to train, validate, export and optimize ATSS model on WGISD public dataset.
+
+.. note::
+
+  To learn how to deploy the trained model, refer to: :doc:`../deploy`.
+
+  To learn how to run the demo and visualize results, refer to: :doc:`../demo`.
+
+The process has been tested on the following configuration.
```

Comment:
Colon?

Reply:
Okay, but I'll fix it in the next documentation PR, if you don't mind